src/index.ts

```diff
@@ -145,7 +147,12 @@ export default class CesiumDnD {
       this._position = pos;
       this._entity = entity;
       this._initialEnableRotate = this.viewer.scene.screenSpaceCameraController.enableRotate;
-      this.viewer.scene.screenSpaceCameraController.enableRotate = false;
+      this._initialEnableTranslate = this.viewer.scene.screenSpaceCameraController.enableTranslate;
+      if (this.viewer.scene.mode === 3) {
+        this.viewer.scene.screenSpaceCameraController.enableRotate = false;
+      } else {
+        this.viewer.scene.screenSpaceCameraController.enableTranslate = false;
+      }
```

Comment:
Since it looks like magic number, this might be better though internally it's same.. Hope this will work. 
```suggestion
      if (this.viewer.scene.mode === Cesium.SceneMode.SCENE3D) {
        this.viewer.scene.screenSpaceCameraController.enableRotate = false;
      } else {
        this.viewer.scene.screenSpaceCameraController.enableTranslate = false;
      }
```

Reply:
updated